language/stdlib/modules/DesignatedDealer.move

```diff
@@ -93,6 +93,22 @@ module DesignatedDealer {
             add_currency<CoinType>(dd, tc_account);
         };
     }
+    spec fun publish_designated_dealer_credential {
+        pragma opaque = true;
```

Comment:
`pragma opaque` is equivalent to `pragma opaque = true`, I would use that.

Reply:
Done.